workspace-tools/src/generators/library-generator/index.ts

```diff
@@ -72,6 +78,7 @@ export default async function (tree: Tree, schema: DhLibrarySchema) {
       LibraryType.environments,
       LibraryType.routing,
       LibraryType.testUtil,
+      LibraryType.shell,
```

Comment:
suggestion: since we haven't used Modules in a while, it's probably worth adding all library types here. I tested with `feature` and `ui` types, and Modules were generated.